### PR TITLE
Remove extraneous 'tags->' in migration

### PR DESCRIPTION
**Pull request by rmarianski**

### CODE REVIEW

File: data/migrations/v1.3.0-point.sql [hunk 1]
```diff
@@ -29,15 +29,15 @@ WHERE
       OR tags -> 'public_transport' IN ('platform', 'stop_area')
       OR tags -> 'railway' IN ('halt', 'level_crossing', 'platform', 'stop',
         'subway_entrance', 'tram_stop')
-      OR tags -> 'tags->icn_ref' = 'true'
```
Comment (nvkelso) [inline data/migrations/v1.3.0-point.sql]: `= 'true'` should be more like `NOT NULL` or `tags ? icn_ref`

File: data/migrations/v1.3.0-polygon.sql [hunk 2]
```diff
@@ -29,15 +29,15 @@ UPDATE
       OR tags -> 'public_transport' IN ('platform', 'stop_area')
       OR tags -> 'railway' IN ('halt', 'level_crossing', 'platform', 'stop',
         'subway_entrance', 'tram_stop')
-      OR tags -> 'tags->icn_ref' = 'true'
```
Comment (nvkelso) [inline data/migrations/v1.3.0-polygon.sql]: same as above...
= 'true' should be more like NOT NULL or tags ? icn_ref

**Comment by nvkelso** [comment]
Approved pending those changes.

File: data/migrations/v1.3.0-polygon.sql (hunk 2 above)
Comment (rmarianski) [inline data/migrations/v1.3.0-polygon.sql]: Updated in b0e4283.

**Pull request merged**